src/helpers/is-supported-runtime.ts

```diff
@@ -0,0 +1,16 @@
+import { detectRuntime } from "./detect-runtime.js";
+
+export function isSupportedRuntime(globalThis: any): boolean {
+  switch (detectRuntime(globalThis)) {
+    case "node": {
+      const [major, minor] = globalThis.process.versions.node
+        .split(".", 2)
+        .map(Number);
+
+      return major >= 22 || (major === 20 && minor >= 17);
+    }
+    default: {
```

Comment:
Shouldn't Deno return true? It should work with the Node compatibility layer out of the box

We should put a baseline version for Deno of 2.0

Reply:
@wolfy1339 IMHO we should only set runtimes as supported if we have tested them in the CI pipeline. Without passing tests, we can not uphold the "contract". But if you want, I can set it to true for now. I mean, right now anyway it is not reliable, as deno and bun mask themselfs as node.

So tell m what you wish. I will adapt.

Reply:
For sure, we should add some tests.
That is the next step.

Even if it returns false, for now, can you add the branch

Reply:
yes

Reply:
done